### PR TITLE
doc: pallet motion

### LINKED ISSUE #42: docs: pallet motion
Please could docs be added to the `motion` pallet at https://github.com/paritytech/extended-parachain-template/blob/main/pallets/motion/README.md, mostly so that any projects using the template can benefit and potentially use as a reference for good documentation practices.

### PR DESCRIPTION
**Pull request by Moliholy**
Closes #42

### CODE REVIEW

File: pallets/motion/README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+
+## Configuration
+
+You can configure `pallet-motion` in combination with `pallet-collective` the following way:
+
+```rust
+impl pallet_motion::Config for Runtime {
+	// ---
+	type SimpleMajorityOrigin =
+		pallet_collective::EnsureProportionMoreThan<AccountId, CouncilCollective, 1, 2>;
```
Comment (Moliholy) [inline pallets/motion/README.md:25]: Should we maybe replace `CouncilCollective` with `pallet_collective::Instance1`? 🤔

Comment (Moliholy) [reply pallets/motion/README.md:25]: Or perhaps

```rust
type CouncilCollective = pallet_collective::Instance1;
impl pallet_motion::Config for Runtime {
// ---
	type SimpleMajorityOrigin =
		pallet_collective::EnsureProportionMoreThan<AccountId, CouncilCollective, 1, 2>;
```

Comment (weezy20) [reply pallets/motion/README.md:25]: I prefer the latter option.

Comment (Moliholy) [reply pallets/motion/README.md:26]: Fixed 👍

**Review by weezy20 — APPROVED** [review]

**Review by stiiifff — APPROVED** [review]

**Pull request merged**